### ISSUE TITLE
Raiden fails to start on first run, succeeds on second

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud-docker#84]
```
xud@xud1-testnet-exchangeunion-com:~$ bash ~/xud.sh
1) Simnet
2) Testnet
3) Mainnet
Please choose the network: 2
Launching testnet environment...
Failed to start service(s): raiden.
Failed to launch testnet environment. For more details, see /home/xud/.xud-docker/xud-docker.log
```

[xud-docker.log](https://github.com/ExchangeUnion/xud-docker/files/3406760/xud-docker.log)

**Comment by kilrau** [comment]
Just reproduced this again. This time the log clearly shows that this is due to parity not being ready yet:
```
docker-compose logs --tail=1000 raiden
Attaching to testnet_raiden_1
^[[36mraiden_1     |^[[0m root
^[[36mraiden_1     |^[[0m config.toml
^[[36mraiden_1     |^[[0m Raiden config exists already
^[[36mraiden_1     |^[[0m Welcome to Raiden, version 0.100.3!
^[[36mraiden_1     |^[[0m ----------------------------------------------------------------------
^[[36mraiden_1     |^[[0m | This is an Alpha version of experimental open source software      |
^[[36mraiden_1     |^[[0m | released as a test version under an MIT license and may contain    |
^[[36mraiden_1     |^[[0m | errors and/or bugs. No guarantee or representations whatsoever is  |
^[[36mraiden_1     |^[[0m | made regarding its suitability (or its use) for any purpose or     |
^[[36mraiden_1     |^[[0m | regarding its compliance with any applicable laws and regulations. |
^[[36mraiden_1     |^[[0m | Use of the software is at your own risk and discretion and by      |
^[[36mraiden_1     |^[[0m | using the software you acknowledge that you have read this         |
^[[36mraiden_1     |^[[0m | disclaimer, understand its contents, assume all risk related       |
^[[36mraiden_1     |^[[0m | thereto and hereby release, waive, discharge and covenant not to   |
^[[36mraiden_1     |^[[0m | sue Brainbot Labs Establishment or any officers, employees or      |
^[[36mraiden_1     |^[[0m | affiliates from and for any direct or indirect liability resulting |
^[[36mraiden_1     |^[[0m | from the use of the software as permissible by applicable laws and |
^[[36mraiden_1     |^[[0m | regulations.                                                       |
^[[36mraiden_1     |^[[0m |                                                                    |
^[[36mraiden_1     |^[[0m | Privacy Warning: Please be aware, that by using the Raiden Client, |
^[[36mraiden_1     |^[[0m | among others, your Ethereum address, channels, channel deposits,   |
^[[36mraiden_1     |^[[0m | settlements and the Ethereum address of your channel counterparty  |
^[[36mraiden_1     |^[[0m | will be stored on the Ethereum chain, i.e. on servers of Ethereum  |
^[[36mraiden_1     |^[[0m | node operators and ergo are to a certain extent publicly available.|
^[[36mraiden_1     |^[[0m | The same might also be stored on systems of parties running Raiden |
^[[36mraiden_1     |^[[0m | nodes connected to the same token network. Data present in the     |
^[[36mraiden_1     |^[[0m | Ethereum chain is very unlikely to be able to be changed, removed  |
^[[36mraiden_1     |^[[0m | or deleted from the public arena.                                  |
^[[36mraiden_1     |^[[0m |                                                                    |
^[[36mraiden_1     |^[[0m | Also be aware, that data on individual Raiden token transfers will |
^[[36mraiden_1     |^[[0m | be made available via the Matrix protocol to the recipient,        |
^[[36mraiden_1     |^[[0m | intermediating nodes of a specific transfer as well as to the      |
^[[36mraiden_1     |^[[0m | Matrix server operators.                                           |
^[[36mraiden_1     |^[[0m ----------------------------------------------------------------------
^[[36mraiden_1     |^[[0m 2019-07-22 11:47:47.523951 [info     ] Starting Raiden                [raiden.ui.runners] architecture=x86_64$
^[[36mraiden_1     |^[[0m
^[[36mraiden_1     |^[[0m Could not contact the Ethereum node through JSON-RPC.
^[[36mraiden_1     |^[[0m Please make sure that the Ethereum node is running and JSON-RPC is enabled.
```

**Comment by kilrau** [comment]
:arrow_up: opened an issue on raiden side. Until then let's delay the start of the raiden container for a couple of seconds (to be tested how long) to make this work @reliveyy

**Comment by kilrau** [comment]
Could you test this again with the new geth setup? @michael1011

**Comment by kilrau** [comment]
Can we use supervisord to only start raiden after geth is up and ready? @reliveyy

**Comment by erkarl** [comment]
Solved by #177
